suse2013/static/css/style.css

```diff
@@ -2187,7 +2189,8 @@ a:hover .package, a:focus .package, a:active .package {
     display: block;
 }
 
-.procedure-title-wrap + .procedure-contents {
+.procedure-title-wrap + .procedure-contents,
+.task-title-wrap + .task-contents {
```

Comment:
Not directly related to this line, but could find a good place for

```
.task-contents .procedure-contents {
  border-left: none;
}
```

That'll remove the extra line still displayed at the side.

Reply:
Ok, see commit 391f98dea50ec